data/packages/com.holyshovelsoft.opensource.log4uni.yml

```diff
@@ -7,7 +7,7 @@ licenseSpdxId: Apache-2.0
 licenseName: Apache License 2.0
 topics:
   - integration
-  - services
+  - debug & log
```

Comment:
Please use validate topics key: "debugging-and-logging"

See more in data/topics.yml